### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
     proxyTable: {}
   },
   backend: {
-    host: process.env.BACKEND_HOST || 'kuzzle',
+    host: process.env.BACKEND_HOST || 'localhost',
```
Comment (xbill82) [inline config/index.js]: Do e2e tests work with this? The name associated to the Kuzzle Proxy is `kuzzle`, in the Docker-compose file.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- docker
+
+env:
+  COMPOSE_VERSION: 1.2.0
+
+before_script:
+  - "curl -sL https://raw.githubusercontent.com/travis-ci/artifacts/master/install | bash"
+
+script:
+  - docker-compose -f docker-compose/test.yml up
```
Comment (xbill82) [inline .travis.yml]: When does it stops?

**Comment by xbill82** [comment]
:+1:

**Comment by xbill82** [comment]
:+1:

**Comment by scottinet** [comment]
+1

**Pull request merged**